cmake/onnxruntime_python.cmake

```diff
@@ -95,11 +95,8 @@ set(onnxruntime_pybind11_state_libs
     onnxruntime_session
     ${onnxruntime_libs}
     ${PROVIDERS_CUDA}
-    ${PROVIDERS_DNNL}
-    ${PROVIDERS_TENSORRT}
     ${PROVIDERS_MIGRAPHX}
     ${PROVIDERS_NGRAPH}
-    ${PROVIDERS_OPENVINO}
     ${PROVIDERS_NUPHAR}
```

Comment:
not directly related to these lines, but you're missing the post build copy step for copying shared libs to the /onnxruntime/capi  , the python package will not work without it.
search for DNNL and TENSORRT below.

Reply:
I see them now, I'm keeping notes for future provider changes as this is easy to forget.